src/ConfigElement/Operator/Concatenator.php

```diff
@@ -46,7 +47,7 @@ public function getLabeledValue($object)
             $value = $c->getLabeledValue($object) ? $c->getLabeledValue($object)->value : null;
 
             if (!$hasValue) {
-                if (!empty($value) || ((method_exists($value, 'isEmpty') && !$value->isEmpty()))) {
+                if (!empty($value) || (is_object($value) && (method_exists($value, 'isEmpty') && !$value->isEmpty()))) {
                     $hasValue = true;
                 }
```

Comment:
An $object is always not empty. So it must be changed to this:
```suggestion
                    if (is_object($value) && method_exists($value, 'isEmpty')) {
                        $hasValue = !$value->isEmpty();
                    } else {
                        $hasValue = !empty($value);
                    }
```